### ISSUE TITLE
0 score on SQR response

### ISSUE THREAD

**Issue by aditydcp** [aditydcp/camapp#2]
Using current version ([last commit](https://github.com/aditydcp/camapp/commit/645bd60327c8c2bc42b870c7197b09378866b962)), calls to SQR Verify API returns a score of 0.0

Possible cause:
- Image is not captured correctly
- Captured image is not sent correctly
- Captured image resolution/size is incompatible with the API process

**Comment by aditydcp** [comment]
**First thought on how to approach this problem**
- [ ] Save the captured image so we can manually try it on the API and look for any difference
- [ ] Configure [my backend server](https://github.com/aditydcp/simple-backend) to display the sent image to see if the image is sent correctly
- [ ] Figure out how to crop the captured image to provide just the code

**Comment by aditydcp** [comment]
Turns out the 0.0 score is caused by a problem on the backend which then giving response with no `score` field.
The 0.0 comes because the 'score' in the in-app Response structure is not-nullable, and since there is no other fields, the response gotten was just as such.

Restructuring the Response data class to be able to take in both the success response and error response shows that the app was getting the same response as how it would if the image is sent manually.

Took a wild guess that the error is caused by the large image, so I tried cropping the image and send it manually and it worked!
Now the app is functional. The photo is cropped before saved and sent so backend wouldn't have problem processing it.
